crates/bitwarden-uniffi/src/tool/mod.rs

```diff
@@ -35,6 +38,18 @@ impl ClientGenerators {
             .passphrase(settings)
             .await?)
     }
+
+    /// **API Draft:** Generate Username
```

Comment:
I assume this isn't a draft any longer?

Reply:
The username and password generators also have the API draft notice in this file, we would probably need to remove those as well. I can change them all on this PR or leave it for another one after.